<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainhub</class>
 <widget class="QDialog" name="mainhub">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
#mainhub {background: rgb(136, 138, 133); }
#topSection { background-color: qlineargradient(spread:reflect, x1:0.5, y1:0, x2:0, y2:0, stop:0 rgba(136, 138, 133, 100), stop:1 rgba(100, 100, 100, 100)); }
QLabel { color: white; }
QLineEdit { border-radius: 10px; }
QPushButton {
    color: white;
    background-color: rgb(85, 87, 83);
    border-width: 0px;
    border-radius: 3px;
	border: 1px solid black;
}
QPushButton:hover { background-color: rgba(100, 100, 100, 1); }
#powerButton {margin: 3px;}
#logOutButton {margin: 3px;}
#restartButton {margin: 3px;}
#settingsButton {margin: 3px;}
#bottomSection{
	border: 1px solid black;
}
#messagesSection{
	border-left: 1px solid black;
}
#recipientSection{
	border-right: 1px solid black;
}
#recipientTitleSection{
	border-bottom: 1px solid black;
}
#recipientTitle{
	font-size: 16px;
	padding: 3px;
	text-transform: uppercase;
}
#message1 {
	padding: 5px;
	border-radius: 5px;
	margin: 5px;
	background-color: rgb(114, 159, 207);
}

#message2{
	padding: 5px;
	border-radius: 5px;
	margin: 5px;
	background-color: rgb(114, 159, 207);
}

#message3{
	padding: 5px;
	border-radius: 5px;
	margin: 5px;
	background-color: rgb(114, 159, 207);
}

#message4{
	padding: 5px;
	border-radius: 5px;
	margin: 5px;
	background-color: rgb(114, 159, 207);
}

#sendMessageButton {
	padding: 5px;
}

#addFriendsButton {
	padding: 5px;
}

#chatsTitle {
	font-size: 25px;

}

#chat1 {
	font-size: 16px;
	padding: 3px;
border:1px solid black

}
#chat2 {
	font-size: 16px;
	padding: 3px;
border:1px solid black

}
#chat3 {
	font-size: 16px;
	padding: 3px;
border:1px solid black

}
#chat4 {
	font-size: 16px;
	padding: 3px;
border:1px solid black
}
#chat5 {
	font-size: 16px;
	padding: 3px;
border:1px solid black

}
#chat6 {
	font-size: 16px;
	padding: 3px;
border:1px solid black
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="topSection" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="currentDate">
        <property name="text">
         <string>Monday, 12-02/2021 14:37 PM</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="userName">
        <property name="text">
         <string>AhmedKasim</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="settingsButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>Settings</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="logOutButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>70</width>
          <height>35</height>
         </size>
        </property>
        <property name="text">
         <string>Log Out</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="restartButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>55</width>
          <height>55</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../Documents/icons/restart.png</normaloff>../Documents/icons/restart.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="powerButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>55</width>
          <height>55</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>../Documents/icons/power.png</normaloff>../Documents/icons/power.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="bottomSection" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>1</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="recipientSection" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>470</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QWidget" name="RecipientButtons" native="true">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>25</height>
            </size>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QPushButton" name="sendMessageButton">
              <property name="text">
               <string>Send Message</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="addFriendsButton">
              <property name="text">
               <string>Add Friend</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="chatsTitle">
           <property name="text">
            <string>Chats</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="chats" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="topMargin">
             <number>1</number>
            </property>
            <item>
             <widget class="QLabel" name="chat1">
              <property name="text">
               <string>LewisActon</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="chat2">
              <property name="text">
               <string>SDI Group</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="chat3">
              <property name="text">
               <string>GeorgeHolland</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="chat4">
              <property name="text">
               <string>TobyHugh</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="chat5">
              <property name="text">
               <string>ElonMusk</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="chat6">
              <property name="text">
               <string>RobertDowneyJr</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="messagesSection" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="topMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QWidget" name="recipientTitleSection" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <property name="topMargin">
             <number>1</number>
            </property>
            <item>
             <widget class="QLabel" name="recipientTitle">
              <property name="text">
               <string>LewisActon</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="horizontalWidget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="topMargin">
             <number>1</number>
            </property>
            <item>
             <widget class="QLabel" name="message1">
              <property name="text">
               <string>Hey Ahmed!</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="message2">
             <property name="text">
              <string>Hey Lewis!</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_8">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QLabel" name="message3">
             <property name="text">
              <string>Want the Answeres for the test?</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <property name="topMargin">
            <number>0</number>
           </property>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="message4">
             <property name="text">
              <string>No! Cheating is bad!</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="messageStatus">
           <property name="text">
            <string>Delivered</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QWidget" name="sendMessageSection" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <property name="topMargin">
             <number>1</number>
            </property>
            <item>
             <widget class="QLineEdit" name="messageInput"/>
            </item>
            <item>
             <widget class="QPushButton" name="sendMessage">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>70</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Send</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>41</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
